src/Microsoft.ML.Dnn/ImageClassificationTransform.cs

```diff
@@ -710,7 +725,7 @@ private void VariableSummaries(RefVariable var)
 
             tf_with(tf.name_scope("train"), delegate
             {
-                var optimizer = tf.train.GradientDescentOptimizer(learningRate);
+                var optimizer = new GradientDescentOptimizerTensor(_learningRateInput);
```

Comment:
> var optimizer = new GradientDescentOptimizerTensor(_learningRateInput); [](start = 15, length = 72)

please pass LR tensor when scheduling is needed.

Reply:
Please upgrade to https://www.nuget.org/packages/TensorFlow.NET/0.11.7

Reply:
Thanks Haiping for quickly adding this constructor!!